editions/fr-FR/tiddlers/$ _config_ViewTemplateTitleFilters_fr-default.tid

```diff
@@ -0,0 +1,7 @@
+created: 20220219141117559
+list-before: $:/config/ViewTemplateTitleFilters/default
+modified: 20220219141157452
+tags: $:/tags/ViewTemplateTitleFilter
+title: $:/config/ViewTemplateTitleFilters/fr-default
+
+[[$:/core/ui/ViewTemplate/title/fr-default]]
```

Comment:
It might be more flexible to use a conditional filter expression such as `[has[fr-title]then[$:/core/ui/ViewTemplate/title/fr-default]]`